server/controllers/Availability.js

```diff
@@ -0,0 +1,96 @@
+const mongoose = require("mongoose");
+const Availability = require("../Models/Availability");
+const asyncHandler = require("express-async-handler");
+
+// @route GET /Availability
+// @desc the Availability list  of the relevant user
+// @access Private
+exports.getAvailability = asyncHandler(async (req, res) => {
+    const userId = req.user.id;
+
+    const availability = await Availability.find({
+        $or: [{ userId: userId }]

```

Comment:
```suggestion
         userId
```

I don't think we need an $or condition as we are only searching for the 'userId' field :)